eyakimovich/01_HTML/Python001.html

```diff
@@ -0,0 +1,71 @@
+ <!DOCTYPE html>  
+ <HTML> 
+ <HEAD>
+ <META charset="utf-8">
+ <TITLE>Python</TITLE>
+ </HEAD>  
+ <BODY> 
+ 
+ <ul><H1>HOW-TO: Программа на Python, Часть 1</H1></ul>
+ 
+  <ul><h3><i>
+  <blockquote>
+<li> Программа на Python — часть 1</li>
+<li> Программа на Python — часть 2</li>
+<li> Программа на Python — часть 3</li>
+ </blockquote>
+  </i></h3></ul>
+  
+ <h2>
+<ul>Содержание</ul>
+<ul>HOW-TO: Программа на Python, Часть 1</ul>
+ </h2>
+<ul>
+<u>
+<blockquote>
```

Comment:
should not be here u and blockquote as a first child inside ul. 
see slides, how looks the lists in html